src/libs/shim-proto/src/empty.rs

```diff
@@ -0,0 +1,166 @@
+// This file is generated by rust-protobuf 2.14.0. Do not edit
+// @generated
```

Comment:
Like `vendor/`, it would be helpful if these files lived in an `auto-generated/` directory I think.